docs-new/docs/graphql/core/actions/index.mdx

```diff
@@ -100,7 +96,7 @@ type UserInfo {
 }
 ```
 
-Read more about `custom types<custom_types>`.
+Read more about [custom types](/graphql/core/actions/types.mdx#custom-types).
```

Comment:
`#custom-types` hash param is not required as this is main page link - not a sub section